e2e/integration/scripts/execute.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+workdir=`pwd`
+cd $workdir
+
+curpath=$PWD
+echo $PWD
+
+if [ ! -d "/var/lib/edged" ]; then
+  sudo mkdir /var/lib/edged && sudo chown $USER:$USER /var/lib/edged
+fi
+
+#run the edge_core and edgecontroller bin to run the E2E
+make edge_core
+make edgecontroller
+sleep 2s
+#Kill the process if it exists
+sudo pkill edgecontroller
+sudo pkill edge_core
+#check the process are killed successfully
+if pgrep edgecontroller >/dev/null
+then
+    echo "edgecontroller process is still Running, Please kill edgecontroller!!"
+    exit 1
+else
+    echo "edgecontroller process is killed"
+fi
+if pgrep edge_core >/dev/null
+then
+    echo "edge_core process is still Running, Please kill edge_core!!"
+     exit 1
+else
+    echo "edge_core process is killed"
+fi
+
+PWD=${curpath}/e2e/integration
+sudo rm -rf $PWD/deployment/deployment.test
+go get github.com/onsi/ginkgo/ginkgo
+sudo cp $GOPATH/bin/ginkgo /usr/bin/
+# Specify the module name to compile in below command
+bash -x $PWD/scripts/compile.sh $1
+export MASTER_IP=121.244.95.60
```

Comment:
We need to find a mechanism to mask the IP address of the master. If our Kubernetes master is public, anyone can use it and our tests may fail due to load. Can we try to find out a way to masks the IP ??